tools/local/monitoring/grafana/provisioning/dashboards/dragonfly.json

```diff
@@ -89,37 +86,40 @@
         "y": 1
       },
       "id": 9,
-      "links": [],
       "maxDataPoints": 100,
       "options": {
         "colorMode": "none",
         "graphMode": "area",
         "justifyMode": "auto",
         "orientation": "horizontal",
+        "percentChangeColorMode": "standard",
         "reduceOptions": {
           "calcs": [
             "lastNotNull"
           ],
           "fields": "",
           "values": false
         },
-        "textMode": "auto"
+        "showPercentChange": false,
+        "textMode": "auto",
+        "wideLayout": true
       },
-      "pluginVersion": "10.1.10",
+      "pluginVersion": "11.3.0",
```

Comment:
can you please make sure it's generated by 10.1.10? I do not want it to be supported only on the latest.

Reply:
should i just change the version number or do i have to import the dashboard in that grafana version and do the adjustments again?

Reply:
adjusted it